fiaisis/frontend
# Add data viewer page

Closes #77, closes #107, closes #108.

## Description

#77: Adds a data viewer page which can be navigated to via the home page or by clicking on a reduction in reduction history.
#107: Adds some extra styling to the instruments page, particularly for light mode.
#108: Renames `Instrument.tsx` to `Instruments.tsx`.

![image](https://github.com/fiaisis/frontend/assets/47388695/f75af713-759e-4df3-9eaf-47520c557dd8)

## How to test the data viewer page

1. Run the API. You must be on the `add_initial_plotting_routes` branch of `FIA-API` (unless it's been merged into main already).
4. `yarn install` and `yarn build` the frontend. Then `yarn run` from `SciGateway` (make sure `settings.json` points to the frontend.
5. In the web browser go to "Reduction history" and click on any entry.
